READMEs/trader.md

```diff
@@ -68,16 +68,21 @@ Copy [`ppss.yaml`](../ppss.yaml) into your own file `my_ppss.yaml` and change pa
 cp ppss.yaml my_ppss.yaml
 ```
 
+Chose the source of the predictions used as signals for trading:
```

Comment:
The default to start should be not from chain.

The "optimize" for from chain should be way down in the "optimize" section.

Same for predictoor.md.

Reply:
Don't use the precious real estate here to tell the user that they can use on chain predictions (let alone show how). There should be no changes in this section.

Rather, much lower in the readme, it shows the user how to use on chain predictions.